.github/ISSUE_TEMPLATE/maintainers-only.yml

```diff
@@ -0,0 +1,38 @@
+name: Maintainers only
```

Comment:
Should any content from the `bug_report.yml` or `feature_request.yml` be moved into new/existing Discussion forms?

Here's the directory for the GitHub Discussion forms: https://github.com/renovatebot/renovate/tree/main/.github/DISCUSSION_TEMPLATE

Right now we have these Discussion forms:

- Ideas & General Discussion
- Mend Renovate App Support

Looks like we're missing a Discussion form for _Self-hosted users_?

Reply:
Maybe call this "Administration only"? I'm not sure most users even know "maintainers"

Reply:
I think "Ideas & General Discussion" can be renamed to "Feature Requests". We can have one "Report a Problem" for bugs, and "Get Config Help" for questions. We can remove "Mend Renovate App Support" because too many people misuse it